### PR TITLE
Allow users to choose which database to use from the frontend

### PR DESCRIPTION
**Pull request by Lev-Stambler**
- Add a modal library component
- Integrate it with the search

### CODE REVIEW

File: apps/api-frontend/src/app/pages/search-pages/search.css
Comment (vitashubin) [inline apps/api-frontend/src/app/pages/search-pages/search.css:8]: nit: styleguide prefers to alphabetize the properties - I'll leave that up to you, but do highly recommend it (it comes in especially handy as apps grow)

File: libs/components/src/lib/modal-chooser-db/modal-chooser.css
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser.css]: nit: stay consistent with using rem

**Review by vitashubin — CHANGES_REQUESTED** [review]
Can you add a link to the site? :) Thanks!

**Comment by Lev-Stambler** [comment]
> Can you add a link to the site? :) Thanks!

I cannot deploy the frontend until it merges into master via Netlify. Here are images though

**Comment by Lev-Stambler** [comment]
Screenshot_2020-07-13_135253
Screenshot_2020-07-13_135337

File: libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx]: prefer to have this Modal have an attached css class, and the style be defined in a .css file

File: libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx]: update?

File: apps/api-frontend/src/app/pages/search-pages/search.css
Comment (Lev-Stambler) [reply apps/api-frontend/src/app/pages/search-pages/search.css:8]: Done (ran through the alphabet a few times in my head :) )

**Review by vitashubin — CHANGES_REQUESTED** [review]
> > Can you add a link to the site? :) Thanks!
> 
> I cannot deploy the frontend until it merges into master via Netlify. Here are images though

Thanks for the images! Mind adding one more for what it looks like when an option is selected?

**Comment by vitashubin** [comment]
> > > Can you add a link to the site? :) Thanks!
> > 
> > 
> > I cannot deploy the frontend until it merges into master via Netlify. Here are images though
> 
> Thanks for the images! Mind adding one more for what it looks like when an option is selected?

Quick note - there's a chrome extension called screencast that lets you take videos of what's happening on your screen and link them to people, might be helpful to have in these situations

**Pull request merged**
